packages/docs/gatsby-node.js

```diff
@@ -0,0 +1,15 @@
+module.exports.createPages = ({ actions }) => {
+  actions.createRedirect({
+    fromPath: '/custom-pragma',
+    isPermanent: true,
+    redirectInBrowser: true,
+    toPath: '/sx-prop',
+  })
+
+  actions.createRedirect({
+    fromPath: '/css',
```

Comment:
Not sure if we wanted to rename `/static-css` to replace this or just leave as-is

Reply:
It's early enough I'd say we can rework URLs without needing redirects, but no real opinion either way.

Reply:
Yeah, I'm not too worried about redirects – could rename the `/static-css` page later if needed